### PR TITLE
Added ability for getting position by mentioning

### LINKED ISSUE #9: Make it possible to see the position of someone else ($position @user-mention)
**Note:** *These sections are recommendations and aren't required. Feel free to add or remove sections depending on the complexity of the feature. If you plan to work with another dev, try to be detailed and clear.*

# Describe the new feature
I just had the idea, that it would be nice to be able to check other users position with $position @user-mention like in the !rank command of mee6, else you would need to use the $leaderboard for it.

## Planning
`message.mentions.users.first()` is for getting the first mentioned users according to [Stack Overflow](https://stackoverflow.com/questions/59252344/how-can-i-get-mentioned-users-username-in-discord-js).
I would look into it in more detail later.

### PR DESCRIPTION
**Pull request by Wissididom**
Fixes #9. Needs to be updated when the bot is switched over to ids.

### CODE REVIEW

**Comment by Wissididom** [comment]
Forgot proof again:
grafik

File: index.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ function checkInput(inputRequest, msg) {
   }
 
   // Display Position
-  if (inputRequest === "position") {
+  if (inputRequest.startsWith("position")) {
 
   leaderBoard.position(msg);
```
Comment (GeorgeCiesinski) [inline index.js:242]: Can you indent line 242 so it is clear it is in the `if` block

File: leaderboard.js [hunk 2]
```diff
@@ -130,19 +130,36 @@ function position(msg) {
...
     const userNames = Array.from(items, user => user[0]);
-  
-    // Find position of message author
-    const userPosition = userNames.indexOf(msg.author.username) + 1;
 
-    if (userPosition === 0) {
-      console.log("User is not on the leaderboard.");
+    const firstMention = msg.mentions.users.first();
+    if (firstMention) { // !== undefined && !== null
+      // Find position of message author
```
Comment (GeorgeCiesinski) [inline leaderboard.js:136]: Please correct this comment as this is finding the position of mentioned user

File: leaderboard.js [hunk 3]
```diff
@@ -251,10 +268,20 @@ function emptyLeaderboard(msg) {
 
 }
 
+function findMember(message, id) {
```
Comment (GeorgeCiesinski) [inline leaderboard.js:271]: Can you add a few comments to findMember and findUser to explain what these do?

File: leaderboard.js [hunk 4]
```diff
@@ -251,10 +268,20 @@ function emptyLeaderboard(msg) {
...
+}
+
 // Exports each function separately
 module.exports.addScore = addScore;
 module.exports.showLeaderboard = showLeaderboard;
 module.exports.position = position;
 module.exports.newChampionship = newChampionship;
 module.exports.dummyLeaderboard = dummyLeaderboard;
 module.exports.emptyLeaderboard = emptyLeaderboard;
+module.exports.findMember = findMember;
```
Comment (GeorgeCiesinski) [inline leaderboard.js:286]: Will these be used elsewhere, or are these functions internal to leaderboard.js?

**Review by GeorgeCiesinski — CHANGES_REQUESTED** [review]
I left a few review comments on the leaderboard.js changes. Can you have a look when you have a sec?

File: index.js (hunk 1 above)
Comment (Wissididom) [reply index.js:242]: Done in a new Pull Request. I don't know why it opened a new one

File: leaderboard.js (hunk 2 above)
Comment (Wissididom) [reply leaderboard.js]: Corrected: changed "message author" to "mentioned user" like you said to do.

File: leaderboard.js (hunk 3 above)
Comment (Wissididom) [reply leaderboard.js:272]: Added these comments

**Review by GeorgeCiesinski — APPROVED** [review]
Changes are approved

File: leaderboard.js (hunk 4 above)
Comment (Wissididom) [reply leaderboard.js]: Removed. We can add them when we really would need them

**Pull request merged**